File: 412_Fizz_Buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let r=[];
    for(let i=1;i<n+1;i++){
        let s='';
        if(i%3===0){
            s+='Fizz';
        }
        if(i%5==0){
            s+='Buzz';
        }
        if(s!==''){
            r.push(s);
        }else{
            r.push((i).toString());
        }
    }
    return r;
};

File: 412.fizz-buzz.py
#
# @lc app=leetcode id=412 lang=python3
#
# [412] Fizz Buzz
#

# @lc code=start
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        foo = []
        for i in range(1, n + 1):
            bar = ''
            if i % 3 == 0:
                bar += 'Fizz'
            if i % 5 == 0:
                bar += 'Buzz'
            if i % 3 != 0 and i % 5 != 0:
                bar = str(i)
            foo.append(bar)
        return foo
# @lc code=end

